app/models/proposal.rb

```diff
@@ -270,19 +264,22 @@ def create_new_observation(user, adder, reason)
     # invalidate relation cache so we reload on next access
     self.observers(true)
     # when explicitly adding an observer using the form in the Proposal page...
-    if adder
-      add_observation_comment(user, adder, reason) unless reason.blank?
+    if adder && reason
+      add_observation_comment(user, adder, reason)
```

Comment:
> before it was `add_observation_comment(user, adder, reason) unless reason.blank?` -- so it would not be called if blank.

Right...I was talking about the next line :wink: 

> I am firmly anti-trailing conditionals!

:+1: